packages/components/pie-textarea/test/helpers/page-objects/selectors.ts

```diff
@@ -0,0 +1,16 @@
+const textArea = {
+    selectors: {
+        container: {
+            dataTestId: 'pie-textarea',
```

Comment:
This is misleading, `container` should refer to the custom element (I appreciate the naming isn't great).

Currently the <pie-textarea> custom element has no data-test-id on it.

As a result:

`page.locator(textArea.selectors.container.dataTestId)` - This would resolve to the custom element

`page.getByTestId(textArea.selectors.container.dataTestId)` - This would resolve to the `textarea` shadow element.

I'd personally suggest renaming this to `pie-textarea-container`.

Then add a new `textArea` property